tests/testthat/test_pipelines.R

```diff
@@ -107,3 +107,52 @@ test_that("survtoregr 3", {
   expect_prediction_surv(p)
   expect_true("distr" %in% p$predict_types)
 })
+
+skip_if_not_installed("mlr3learners")
+
+test_that("survtoclassif", {
+  requireNamespace("mlr3learners")
+
+  pipe = mlr3pipelines::ppl("survtoclassif", mlr3learners::LearnerClassifLogReg$new())
```

Comment:
better test with `GraphLearner` option, easier for assertions and that's the end use case we want (since we will test with CV)